### ISSUE TITLE
Warnings

### ISSUE THREAD

**Issue by emily1822** [marik22354/Facebook-Sender_V2#4]
The tool get somewhat flagged by AV vendors, but once I've tested in a sandbox, I can confirm its all good, but sometimes it buggy and wont spit out the code for OTPs, for the other features it work good

**Comment by marik22354** [comment]
Yes that's defenetively happening becouse facebook is using too much security nowaday, I've implemented all needed things to avoid getting banned
